languages/csharp/reference/README.md

```diff
@@ -160,25 +160,25 @@ While C# is primarily an OO language, lots of functional concepts have been adde
-| `enums-basic`               | Know of the existence of the `enum` keyword. Know how to define enum members. Know how to assign values to enum members. Know how to get an enum's numeric value. Know how to convert a `string` to an `enum` and vice versa.                             |
-| `loops-while`               | Know how to write a `while` loop.                                                                                                                                                                                                                         |
-| `numbers-basic`             | Know of the existence of the two most commonly used number types, `int` and `double`, and understand that the former represents whole numbers, and the latter floating-point numbers. Know of basic operators such as multiplication and comparison.      |
-| `numbers-floating-point`    | Know of the existing of the three floating point types: `double`, `float` and `decimal`. Know when to use which type.                                                                                                                                     |
-| `strings-basic`             | Know of the existence of the `string` type. Know how to create a string. Know of some basic methods (like finding the index of a character in a string, or returning a part the string). Know how to do basic string formatting.                          |
-| `time-basic`                | Know of the existence of the `DateTime` type. Know of the individual, time-related properties.                                                                                                                                                            |
-| `type-conversion-numbers`   | Know how to convert from one numeric type to another. Know what implicit and explicit conversions are.                                                                                                                                                    |
+| concept                   | interpretation                                                                                                                                                                                                                                            |
+| ------------------------- | --------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
+| `arrays-basic`            | Know of the existence of the `Array` type. Know how to define an array. Know how to access elements in an array by index. Know how to iterate over elements in an array. Know of some basic functions (like finding the index of an element in an array). |
```

Comment:
I think something went wrong here, as this list has the older concept names (`-basic` suffix, etc.). For the latest version, see: https://github.com/exercism/v3/blob/master/languages/csharp/reference/README.md